### ISSUE TITLE
Upgrade Filebeat to v7.8.1

### ISSUE THREAD

**Issue by toszo** [epiphany-platform/epiphany#846]
Filebeat should be updated to at least 7.2.0 since this version contains fix for "registry entries not being cleaned due to race conditions." - Installation not deterministically failed on Kubernetes master and worker nodes. Temporary solution was to disable "system" module. 

https://www.elastic.co/guide/en/beats/libbeat/current/release-notes-7.2.0.html

**Comment by to-bar** [comment]
We need to adjust automated tests (issue #1657).

**Comment by to-bar** [comment]
Testing upgrades is blocked by #1647.

**Comment by rpudlowski93** [comment]
@to-bar Shouldn't we consider upgrading filebeat to the lastest version 7.9.2? For AKS/EKS I already added filebeat as daemonset in the latest version 7.9.2 ( don't want to add new feature in old version ). It could be nice to have the same version in both scenarios.

**Comment by rafzei** [comment]
> @to-bar Shouldn't we consider upgrading filebeat to the lastest version 7.9.2? For AKS/EKS I already added filebeat as daemonset in the latest version 7.9.2 ( don't want to add new feature in old version ). It could be nice to have the same version in both scenarios.

We will upgrade elasticsearch-oss to version 7.9.1 (#1719) as we have to upgrade ODFE to 1.10.1 ([7.9.1 is 1.10.1](https://opendistro.github.io/for-elasticsearch-docs/version-history/))
AFAIK its not recomended to have version mismatch so we should upgrade filebeat-oss to 7.9.1 as well.

**Comment by to-bar** [comment]
IMHO in this case we can upgrade Filebeat to 7.9.2 (having Elasticsearch 7.9.1) since according to [7.9.2 release notes]( https://www.elastic.co/guide/en/beats/libbeat/7.9/release-notes-7.9.2.html) it contains one nice to have [bugfix](https://github.com/elastic/beats/pull/21048) and other changes seem to be acceptable for Epiphany.

**Comment by rafzei** [comment]
> 
> 
> IMHO in this case we can upgrade Filebeat to 7.9.2 (having Elasticsearch 7.9.1) since according to [7.9.2 release notes](https://www.elastic.co/guide/en/beats/libbeat/7.9/release-notes-7.9.2.html) it contains one nice to have [bugfix](https://github.com/elastic/beats/pull/21048) and other changes seem to be acceptable for Epiphany.

It's fine, I will update the issue description.

**Comment by to-bar** [comment]
Related issue to test together: #1774

**Comment by to-bar** [comment]
Test OK (verified on Ubuntu and RHEL)
